app/controllers/v0/letters_controller.rb

```diff
@@ -14,11 +14,11 @@ def index
       end
     end
 
-    def show
+    def download
       unless EVSS::Letters::Letter::LETTER_TYPES.include? params[:id]
         raise Common::Exceptions::ParameterMissing, 'letter_type', "#{params[:id]} is not a valid letter type"
       end
-      response = service.download_letter_by_type(params[:id])
+      response = service.download_by_type(params[:id], request.body.string)
```

Comment:
Unrelated to this PR: does `id` represent the ID of the letter to download?  I just want to make sure we're validating that one user may not download the letters of another user.